### DISCUSSION TITLE
Show Datapoint on Timeseries Chart

### DISCUSSION THREAD

**Discussion by stewam7** [grafana/grafana#71102] [Dashboards, Panels & Visualizations]
Hello,

IHAC looking for the ability to display a datapoint on a time series chart. Similar to what we are able to do with a bar chart. This community post I believe is also looking for the same:

https://community.grafana.com/t/any-way-to-show-values-on-graphs/24899

Thanks!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82600. Feel free to continue the discussion around this there. Thank you!
